### PR TITLE
migrate timer_proxy part 2

### PR DESCRIPTION
**Pull request by chkeita**
Continuing the migration of time_proxy

### CODE REVIEW

File: src/ApiService/ApiService/onefuzzlib/Auth.cs [hunk 1]
```diff
@@ -0,0 +1,16 @@
+﻿namespace Microsoft.OneFuzz.Service;
+
+using System.Security.Cryptography;
+
+public class Auth
+{
+    public static Authentication BuildAuth()
+    {
```
Comment (stishkin) [inline src/ApiService/ApiService/onefuzzlib/Auth.cs:8]: you verified it matches Python calling ssh-key gen and it matches ?

File: src/ApiService/ApiService/onefuzzlib/Network.cs [hunk 2]
```diff
@@ -50,7 +44,7 @@ public static async Async.Task<Network> Create(string region, ICreds creds, ICon
             }
             else
             {
-                var networkId = GenerateGuidv5(NETWORK_GUID_NAMESPACE, string.Join("|", networkConfig.AddressSpace, networkConfig.Subnet));
+                var networkId = Faithlife.Utility.GuidUtility.Create(NETWORK_GUID_NAMESPACE, string.Join("|", networkConfig.AddressSpace, networkConfig.Subnet), 5);
```
Comment (stishkin) [inline src/ApiService/ApiService/onefuzzlib/Network.cs:47]: we need to take a 3rd party dependency to generate network id ?

File: src/ApiService/ApiService/onefuzzlib/Auth.cs (hunk 1 above)
Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/Auth.cs:8]: No i did not. I am counting on check pr to validate this

File: src/ApiService/ApiService/onefuzzlib/Network.cs (hunk 2 above)
Comment (chkeita) [reply src/ApiService/ApiService/onefuzzlib/Network.cs:47]: Yes, There is not way generate a deterministic Guid with the base library. 
If you have a better suggestion let me know

**Review by stishkin — APPROVED** [review]

**Pull request merged**